app/assets/stylesheets/hubstats/bootstrap.css

```diff
@@ -654,7 +654,8 @@ blockquote .small:before {
   content: '\2014 \00A0';
```

Comment:
I think it's best practice not to modify the boostrap.css file. That way when bootstrap bumps the version we can just replace this file with the new version.